### DISCUSSION TITLE
Schema results not loading

### DISCUSSION THREAD

**Discussion by radhay-samagra** [hasura/graphql-engine#8591] [General] [answered]
I am using Hasura in many of my projects. In one of my projects, I am using v2.0.6 and when I log in to the console, I cannot fetch results from my Postgres tables as it keeps loading forever and also causing locks on tables. But the graphql and REST queries are working fine. Can anyone help me with the issue?

**Comment by meetzaveri** [comment, marked as answer]
@radhay-samagra 

Hello 👋🏻 

Can you let us know that if you are using hasura cloud (cloud version) or hasura CE (open source version) for this specific project?
